### ISSUE TITLE
Translated field becomes null after deploying any functions

### ISSUE THREAD

**Issue by amin79** [firebase/extensions#1838]
Any time I deploy my functions into firebase, all the translated fields become NULL.

`firebase/firestore-translate-text@0.1.15`

**Comment by cabljac** [comment]
Hi, can you share your extension config (redacting anything sensitive of course)

**Comment by amin79** [comment]
Hello,

the picture is attached. Please let me know if you need more.
I have installed this extension 3 times for 3 different fields. 2 of them in one collections. All with the same configs (No advanced configs). When I deploy my functions, I see that one field became null and the other not. Some times both of them are null.

<img width="634" alt="Screenshot 2023-11-24 at 18 33 20" src="https://github.com/firebase/extensions/assets/33101184/fa596c52-c5e9-43fa-9e05-c80c83c8ae50">

**Comment by cabljac** [comment]
Thanks, looking into this now.

**Comment by cabljac** [comment]
Ah you have multiple instances of the same extension, watching the same collection?

**Comment by cabljac** [comment]
<img width="897" alt="Screenshot 2023-11-24 at 18 18 09" src="https://github.com/firebase/extensions/assets/32874567/e417a777-0c78-4939-80ec-5c2df5f55e36">

**Comment by amin79** [comment]
Yes . 2 for one collection and one for another one. You see the data because I updated the texts and they are created again. Ok. Once I deploy the functions again and let you know

**Comment by cabljac** [comment]
Hi there, any update on this? Any logs?

**Comment by amin79** [comment]
Hi. Today morning I tested it again. The translated field on one of the documents was NULL (I have  2 translated fields. One has translated text but the other doesn't). I checked it after deploying new functions. But I'm not sure if it was NULL before of after deploy became NULL.

BTW, why is it null? Even before or after deploying.  It has to have translated field. It's a bit hard to reproduce the problem.

One more question. Does this extension translate everything or if it's a none sense text (just typing like "sdfsdfh") it returns null?  Because in another document I has such a text and it also was not translated
